tests/integration/cluster-users/sar-test.sh

```diff
@@ -48,31 +48,111 @@ function __deleteTestNamespace() {
 }
 
 function __createRoleBindingForNamespaceDeveloper() {
+	local result=1
 	set +e
 	kubectl create rolebinding 'namespace-developer' --clusterrole='kyma-developer' --user="${DEVELOPER_EMAIL}" -n "${CUSTOM_NAMESPACE}"
-	result = $?
+	result=$?
 	set -e
 
-	if [ result -eq 0 ]; then
+	if [[ ${result} -eq 0 ]]; then
 		echo "----> PASSED"
 		return 0
 	fi
 
 	echo "----> |FAIL|"
+	return 1
 }
 
 function __createNamespaceForNamespaceAdmin() {
+	local result=1
 	set +e
 	kubectl create namespace "${CUSTOM_NAMESPACE}"
-	result = $?
+	result=$?
 	set -e
-  
-	if [ result -eq 0 ]; then
+
+	if [[ ${result} -eq 0 ]]; then
 		echo "----> PASSED"
 		return 0
 	fi
-  
+
 	echo "----> |FAIL|"
+	return 1
+}
+
+function __testPermissions() {
+	local OPERATION="$1"
+	local RESOURCE="$2"
+	local TEST_NS="$3"
+	local EXPECTED="$4"
+	local TEST="not-set-yet"
+
+	if [[ "${TEST_NS}" != "--all-namespaces" ]]; then
+		TEST_NS="-n${TEST_NS}"
+	fi
+
+	sleep 0.1
```

Comment:
Is this really required?

Reply:
I added this to reduce the stress on api-server during test execution. It extends test duration by just few seconds in total.